cli.sh

```
@@ -1388,7 +1388,29 @@ copy_pi_configs() {
+		local pi_settings="$HOME/.pi/agent/settings.json"
+		local fusion_packages='["npm:@tintinweb/pi-subagents","npm:pi-permission-system"]'
+		if ! command -v jq >/dev/null 2>&1; then
+			log_warning "Pi settings.json exists but jq is unavailable; install @tintinweb/pi-subagents and pi-permission-system manually to enable Fusion agents"
+		elif jq -e --argjson required "$fusion_packages" '(.packages // []) as $packages | $required | all(. as $package | $packages | index($package) != null)' "$pi_settings" >/dev/null 2>&1; then
+			log_info "Pi settings.json already includes Fusion subagent support"
+		elif [ "$DRY_RUN" = true ]; then
+			log_info "[DRY RUN] Would add Fusion subagent support to existing Pi settings"
+		else
+			local merged_settings
+			merged_settings=$(execute_quoted mktemp "$HOME/.pi/agent/.settings.json.my-ai-tools.XXXXXX") || return 1
+			if execute_quoted jq --argjson required "$fusion_packages" '(.packages // []) as $packages | .packages = (reduce $required[] as $package ($packages; if index($package) then . else . + [$package] end))' "$pi_settings" >"$merged_settings"; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)